source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -239,12 +237,9 @@ namespace RendererQml
                         context->addToShowCardLoaderComponentList(componentId, showCardAction);
 
                         //Add show card loader to the parent container
-                        if (addLoaderSeparator)
-                        {
-                            AddSeparator(uiContainer, std::make_shared<AdaptiveCards::Container>(), context);
-                            addLoaderSeparator = false;
-                        }
-                        auto uiLoader = std::make_shared<QmlTag>("Loader");
+                        AddSeparator(uiContainer, std::make_shared<AdaptiveCards::Container>(), context, true, uiAction->GetId() + "_loader");
```

Comment:
Can we not add the separator to the component instead?

Reply:
I have tried adding the separator in component in the outer most rect as well as inner rect (body layout) it then disturbs the background color of the card and also the top margin too
so went with this option instead